core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -79,6 +81,18 @@ private MobileCore(Context context, Options options)
         } else {
             this.httpLayer = options.httpServiceModule;
         }
+
+        // -- init and send default metrics
+        initMetrics();
+    }
+
+    private void initMetrics() {
+        MetricsService metrics = getInstance(MetricsService.class);
+        metrics.sendDefaultMetrics();
```

Comment:
[Minor] In proposal this is linked as `app` group 
https://github.com/aerogear/proposals/blob/master/metrics/default-mobile-app-metrics.md